webrecorder/browsertrix
# Allow users to change display name and email

Resolves https://github.com/webrecorder/browsertrix-cloud/issues/522



### Changes

- Refreshes account settings page styles
- Users can change their display name
- Users can change their email address, with verification status shown (note: backend does not automatically send new verification email)

### Manual testing

1. Log in and click app bar user dropdown -> Account Settings
2. Enter in a new display name and click "Save". Verify name saves as expected
3. Enter in a new email address and click "Save". Verify email saves as expected and verification status is updated
4. Click "Change Password". Verify password input receives focus
5. Enter in an incorrect current password. Verify form resets and does fails to save
6. Enter in correct current password and new password. Verify password saves as expected and form is hidden again.

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| Account Settings | <img width="666" alt="Screenshot 2023-10-10 at 4 32 13 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/955a3f99-6917-4cef-85ce-1bb241be5a5b"> |
| Account Settings - Email (unverified) | <img width="685" alt="Screenshot 2023-10-10 at 4 39 18 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/627bbc67-a513-4154-b3a9-7ad92764cc27"> |
| Account Settings - Password (expanded) | <img width="664" alt="Screenshot 2023-10-10 at 4 32 21 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/307c4d99-5dc8-450b-bcbd-664982b1fdf3"> |



### Follow-ups
Strong password requirements are not enforced, will be handled in https://github.com/webrecorder/browsertrix-cloud/issues/1233
